lib/core-events/src/index.ts

```diff
@@ -39,8 +39,12 @@ enum events {
   NAVIGATE_URL = 'navigateUrl',
 }
 
+const symbols = {
```

Comment:
why symbol? @ghengeveld

Reply:
can't we just make it a string for consistency?

Reply:
Changed it to an Error. String isn't going to work because many libs naively expect only Errors to ever get thrown. So when they try to read the message property it blows up.